### ISSUE TITLE
pictures unrelated to US-RSE?

### ISSUE THREAD

**Issue by cmaimone** [USRSE/gallery#4]
I'm not sure why pictures that don't seem to have a relation to US-RSE/RSEs in general are included here -- for example, multiple photos of food.  Seems like it should be limited to related media

**Comment by vsoch** [comment]
It's from our eat-street channel!

**Comment by cmaimone** [comment]
I think that's a fun thing to share in Slack -- not sure that everything
needs to go to the gallery though.  Makes it less useful for finding
RSE-related images that people are OK with having used for presentations,
websites, etc.

**Comment by vsoch** [comment]
Yeah it would work to remove, although I'm not sure we'd have a ton of pictures left! We could also make a separate eat-street gallery, or just delete. I'd like this page to perhaps be something like a document archive so it would warrant a rename and a bit of a refactor anyway, happy to do some change that others agree with is warranted.
